roles/metropolregion_termine/templates/metropol_run.sh.j2

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
@michaelu123  dieses Skript wird immer um 23:17 Uhr aufgerufen. Die Dinge in den geschweiften Klammern sind Platzhalter die durch Ansible ersetzt werden.